<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTicket</class>
 <widget class="QMainWindow" name="CreateTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fill in the fields to create a ticket</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_id_flight">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flight ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_field_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_seat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_time">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_origin">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_date">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_origin">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_destiny">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destiny</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_field_destiny">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_box_number_seat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>900</number>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_box_id_flight">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_ticket">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="time_flight">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_cpf">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_field_cpf">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>999.999.999-99</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>251</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_create">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
